src/Native/Unix/System.Native/pal_termios.h

```diff
@@ -0,0 +1,22 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+include "pal_types.h"
+
+/**
+ * Get states of serial line.
+ */
+extern "C" int32_t SystemNative_Terminal_GetDcd(int fd);
```

Comment:
System.IO.Ports is standalone package, but this is adding dependency on System.Native. System.Native is internal implementation detail. We do not want things outside .NET Core distro to depend on it.

Should System.IO.Ports have its own supporting native library instead? Or depend on Mono.Posix?